### ISSUE TITLE
Delete operation agenst the feed

### ISSUE THREAD

**Issue by stianvillanger** [rackerlabs/atom-hopper#295]
Hi

Thanks for the great atom tool Atom Hopper.

We're using Atom Hopper as a message feed at work, and want to remove/delete ‘old’ entries that are invalid. 

I am trying to invoke a HTTP DELETE on:

https://domain.com/namespace/feed/entries/urn:uuid:9b850562-d357-4cf8-8811-048a6730e869

with response:

java.lang.UnsupportedOperationException: Not supported.

Is there an other way to perform a delete operation against the feed?

Best regards
Stian

**Comment by usnavi** [comment]
Hi,

What adapter are you using?  I don't think its supported by any of the adapters.  We use the JDBCAdapter, and that method throws and UnsupportedOperationException, which I believe you are seeing:

https://github.com/rackerlabs/atom-hopper/blob/master/adapters/jdbc/src/main/java/org/atomhopper/jdbc/adapter/JdbcFeedPublisher.java#L306

Greg

**Comment by stianvillanger** [comment]
Correct, not only the JDBC adapter but all of them return UnsupportedOperationException("Not supported."); on HTTP delete.
